concepts/observability-strategy/configurable-tracing/pocs/w3c-tracecontext/README.md

```diff
@@ -0,0 +1,34 @@
+## Goal
+
+Prove that istio can be switched to w3c-tracecontext protocol and verify that Kyma components will support it native
+
+## Strategy
+
+At the moment, the only solution to use w3c-tracecontext already is via an openCensus tracer for istio. As Jaeger does not support the openCensus protocol, an otel-collector needs to be introduced additionally which does the conversion to the zipkin/jaeger protocol. As the goal anyway is to introduce an otel-collector, that approach sournds feasible as the central otel-collector could support openCensus temporarily as additional receiver.
```

Comment:
```suggestion
At the moment, the only solution to use w3c-tracecontext is with an OpenCensus tracer for Istio. Because Jaeger does not support the OpenCensus protocol, an otel-collector must be introduced additionally, which converts the data to the Zipkin/Jaeger protocol. As the goal anyway is to introduce an otel-collector, this approach is feasible because the central otel-collector could support OpenCensus temporarily as additional receiver.
```